docs/versioning/how-to-version.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+nav_order: 4
+title: How to create versions  
+has_children: false
+parent: Versioning
+---
+# How to create versions in the Registry
+
+## Step 1. Navigate to the item record in the Registry
+
+- Ensure you're logged in.
+- Navigate to the item record in the Registry
+- Click on the "VERSIONING" tab as shown below
+
+|                         Versioning Tab                                                                 |
+| :----------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-versioning-tab.jpg" alt="drawing" width="600"/> |
+
+
+
+## Step 2. Create a new version
+
+- Click on the "Create New Version" button to create a new version of the item.
```

Comment:
Same as dataset - need the correct permissions + latest version

Reply:
added pre-requisites and info about needing to be at the latest version